src/addrman.h

```diff
@@ -569,13 +572,13 @@ friend class CAddrManTest;
 
         // Prune new entries with refcount 0 (as a result of collisions).
         int nLostUnk = 0;
-        for (std::map<int, CAddrInfo>::const_iterator it = mapInfo.begin(); it != mapInfo.end(); ) {
+        for (MapInfo::const_iterator it = mapInfo.begin(); it != mapInfo.end(); ) {
```

Comment:
nit:
```suggestion
        for (auto it = mapInfo.cbegin(); it != mapInfo.cend(); ) {
```

Reply:
Done. Also, skipping the mention of `MapInfo` here makes it used in just one place. Dropped it and used `std::unordered_map<...` directly since there is no repetition of the tedious `std::unordered_map<...` now. Thanks!